lib/src/rules/prefer_relative_imports.dart

```diff
@@ -85,6 +66,7 @@ class _Visitor extends SimpleAstVisitor<void> {
     String importUri = node?.uri?.stringValue;
     if (importUri == null) return false;
 
+    // todo (pq): is there a better way to do this?
```

Comment:
Ah, to determine the uri w/o relying on trying to parse and catching exceptions?

Reply:
Yeah.  It seems like a bunch of work when I'd expect the URI to already be around.

Moreover, if all we want is to know if it's package-qualified we might simplify further by just testing something like `importUri.startsWith('package:')`.  🤔